autogtp/Production.cpp

```diff
@@ -37,16 +37,18 @@ void ProductionWorker::run() {
              game.readMove();
          } while (game.nextMove());
          QTextStream(stdout) << "Game has ended." << endl;
+         int movesMade = 1;
```

Comment:
The way this value is set as the default makes me suspicious the error handling around here is wonky. Notably, should we call emit resultReady when we can't do a getScore()?  Ignoring that problem (which exists in the current code), can't we already just call game.getMoveNum() here?